<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_page</class>
 <widget class="QWidget" name="main_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>900</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>20</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::tab-bar {
   border: 1px solid #3F3838;
}

QTabBar::tab {
  background: #3F3838;
  color: white;
  padding: 10px;
 }

 QTabBar::tab:selected {
  background: #3F3838;
  color : #FFA800;
 }

</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <widget class="QWidget" name="dash_board">
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background : qlineargradient(x2:1, y2:1, x0:1, y0:1,
                stop:0 rgb(255,255,224), stop: 0.4 rgba(255,251,219, 40), stop:1 rgb(235,119,20));</string>
    </property>
    <attribute name="title">
     <string>DASHBOARD     </string>
    </attribute>
    <attribute name="toolTip">
     <string>Your dashboard</string>
    </attribute>
    <widget class="QWidget" name="heart_rate_widgets" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>711</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px solid black;
border-radius : 15px;
background : white;</string>
     </property>
     <widget class="MplWidget" name="graph_widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>671</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_1day">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>1 day</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7days">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>7 days</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_1month">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>1 month</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3month">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>3 month</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_extend">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>EXTEND</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="steps_widgets" native="true">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>30</y>
       <width>711</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px solid black;
border-radius : 15px;
background : white;</string>
     </property>
    </widget>
    <widget class="QWidget" name="food_widgets" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>430</y>
       <width>711</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px solid black;
border-radius : 15px;
background : white;</string>
     </property>
    </widget>
    <widget class="QWidget" name="quote_widgets" native="true">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>430</y>
       <width>711</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px solid black;
border-radius : 15px;
background : white;</string>
     </property>
     <widget class="QLabel" name="quote_image">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>620</width>
        <height>290</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 0px;
background-image: url(./image/quote_1.jpg);
background-size: cover;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="profile">
    <property name="styleSheet">
     <string notr="true">background : qlineargradient(x2:1, y2:1, x0:1, y0:1,
                stop:0 rgb(255,255,224), stop: 0.4 rgba(255,251,219, 40), stop:1 rgb(235,119,20));</string>
    </property>
    <attribute name="title">
     <string>PROFILE</string>
    </attribute>
    <widget class="QWidget" name="profile_widget_1" native="true">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>80</y>
       <width>620</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px solid black;
border-radius : 15px;
background : white;</string>
     </property>
     <widget class="QLabel" name="profile_information_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>25</y>
        <width>321</width>
        <height>67</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;</string>
      </property>
      <property name="text">
       <string>Profile Information</string>
      </property>
     </widget>
     <widget class="QLabel" name="profile_picture">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Font Awesome 5 Free Regular</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;</string>
      </property>
      <property name="text">
       <string>ï€‡</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SimSun</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;
</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>380</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 0px;
border-style : groove;
border-width : 1.5px;
border-color : rgb(128,128,128);</string>
      </property>
     </widget>
     <widget class="QLabel" name="email_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SimSun</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;
</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>380</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 0px;
border-style : groove;
border-width : 1.5px;
border-color : rgb(128,128,128);</string>
      </property>
     </widget>
     <widget class="QLabel" name="email_picture">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Font Awesome 5 Free Regular</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;</string>
      </property>
      <property name="text">
       <string>ïƒ </string>
      </property>
     </widget>
     <widget class="QLabel" name="password_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>400</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SimSun</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;
</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>450</y>
        <width>380</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 0px;
border-style : groove;
border-width : 1.5px;
border-color : rgb(128,128,128);</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_picture">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>440</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Font Awesome 5 Free Regular</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;</string>
      </property>
      <property name="text">
       <string>ï”‚</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="profile_widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>852</x>
       <y>80</y>
       <width>620</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px solid black;
border-radius : 15px;
background : white;</string>
     </property>
     <widget class="QLabel" name="birth_date_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SimSun</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;
</string>
      </property>
      <property name="text">
       <string>Birth-date</string>
      </property>
     </widget>
     <widget class="QLabel" name="weight_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SimSun</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;
</string>
      </property>
      <property name="text">
       <string>Weight</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="weight_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 0px;
border-style : groove;
border-width : 1.5px;
border-color : rgb(128,128,128);</string>
      </property>
      <property name="inputMask">
       <string>999</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="birth_date_picture">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Font Awesome 5 Free Regular</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;</string>
      </property>
      <property name="text">
       <string>ïžƒ</string>
      </property>
     </widget>
     <widget class="QLabel" name="weight_picture">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Font Awesome 5 Free Regular</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;</string>
      </property>
      <property name="text">
       <string>ï’–</string>
      </property>
     </widget>
     <widget class="QLabel" name="height_label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>260</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SimSun</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;
</string>
      </property>
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="height_lineEdit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>310</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 0px;
border-style : groove;
border-width : 1.5px;
border-color : rgb(128,128,128);</string>
      </property>
      <property name="inputMask">
       <string>999</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="height_picture">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>290</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Font Awesome 5 Free Regular</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;
background-image : url(&quot;./image/height_picture.png&quot;);
</string>
      </property>
      <property name="text">
       <string>ï•ˆ</string>
      </property>
     </widget>
     <widget class="QLabel" name="gender_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>400</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SimSun</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;
</string>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
     <widget class="QLabel" name="weight_picture_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>430</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Font Awesome 5 Free Regular</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : none;</string>
      </property>
      <property name="text">
       <string>ïˆ¨</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_Button">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>519</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color : white;
background : #1347EC;
border-radius : 20px;
</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="birth_dateEdit">
      <property name="geometry">
       <rect>
        <x>139</x>
        <y>160</y>
        <width>301</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : black;
border-style: solid;
border-width: 1px;
border-radius : 0px;
border-color : rgb(128,128,128);</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="sex_comboBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>450</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border : black;
border-style: solid;
border-width: 1px;
border-radius : 0px;
border-color : rgb(128,128,128);</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="user_name">
    <attribute name="title">
     <string>                                                                                                        Sarin Wanichwasin</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>